packages/itwinui-css/src/stepper/stepper.scss

```diff
@@ -17,101 +15,91 @@
 }
```

Comment:
I know its not connected, but that bottom margin should be there only for the `long` type with bottom description (so maybe we could move this there as margin top...).
Just saying cause we had issues with hardcoded margins before (like in toggle switch)